src/main/blackbox/blackbox.c

```diff
@@ -890,12 +905,12 @@ static void writeInterframe(void)
 
 #ifdef USE_SERVOS
     if (testBlackboxCondition(CONDITION(SERVOS))) {
-        STATIC_ASSERT(ARRAYLEN(servo) <= 8, "TAG8_8SVB supports at most 8 values"); 
+        STATIC_ASSERT(ARRAYLEN(servo) <= 8, "TAG8_8SVB supports at most 8 values");
         int32_t out[ARRAYLEN(servo)];
         for (unsigned x = 0; x < ARRAYLEN(servo); ++x) {
             out[x] = blackboxCurrent->servo[x] - blackboxLast->servo[x];
         }
-        
+
```

Comment:
your editor cleaned this as well. while it needs to be cleaned up, maybe not this PR.

Reply:
@haslinghuis , thoughts?
maybe empty line okay, but not code-lines?
@ledvinap ?

Reply:
I am not seeing github warning about uncompatible with master branch.
My current "attitude log" PR has removed white spaces already.

Reply:
If PR can merge without conflicts then why not?

Reply:
There is no conflict until first one is merged. And maybe giot can resolve this trivial case automatically

Reply:
Okay for this one. For bigger PR's it is recommended to split in refactoring and improvement PR's to ease review.

Reply:
> There is no conflict until first one is merged. And maybe git can resolve this trivial case automatically

Do i understand true?
I will merge first and all are good. But other openned PRs, what have previous white spaces, will have merge conflict probably?

Reply:
Exactly.